zips/draft-ecc-lockbox-disbursement.md

```diff
@@ -0,0 +1,245 @@
+Deferred Dev Fund Lockbox to a 3-of-5 P2SH multisig with keys held by the same
+Key-Holder Organizations as above, starting at block height
+$\mathsf{activation\_height} + N$ and continuing at periodic intervals of $N$
+blocks until $\mathsf{stream\_end\_height}$. Each disbursement empties the
+lockbox. 
+
+This is equivalent to specifying
+$\frac{\mathstrut\mathsf{stream\_end\_height} \,-\, \mathsf{activation\_height}}{N}$ [One-time lockbox disbursement]s, 
+that all output to the same address.
+
+#### Rationale for periodic disbursement
+
+Classic funding streams [^zip-0207] produce many small output values, due to
+only being able to direct funds from a single block's subsidy at a time. This
+creates operational burdens to utilizing the funds — in particular due to block
+and transaction sizes limiting how many outputs can be combined at once, which
+increases the number of required transactions and correspondingly the overall
+fee.
+
+The periodic lockbox disbursement mechanism can produce the same effective
+funding stream, but with aggregation performed for free: the output to the
+funding stream recipient is aggregated into larger outputs every $N$ blocks. In
+the specific case of Mechanism 2b, the recipient multisig address would receive
+around 40 outputs, instead of around 1,300,000.
+
+# Privacy and Security Implications
+
+As development funding is a public good on the Zcash network, there are not
+relevant privacy concerns related to this proposal; all disbursement of
+(but not necessarily subsequent distribution) of development funds is 
+transparent and auditable by any participant in the network.
+
+#### Security implications of the One-Time Lockbox Disbursement
+
+After the activation block of this ZIP has been mined, all development funds
+previously accrued to the in-protocol lockbox will be held instead by a 3-of-5
+multisig address. The key-holders for this address will have the capability to
+spend these funds. Compromise or loss of 3 of these 5 keys would result in
+total loss of funds; as such, in the event of the compromise or loss of a
+single key, the Key-Holders MUST establish a new multisig key set and address,
+and transfer remaining unspent funds held by the original address before
+additional loss or compromise occurs.
+
+Because this is a one-time disbursement, additional key rotation infrastructure
+is not required.
+
+#### Security implications for Option 1
+
+Funds will continue to securely accrue to the Deferred Development Lockbox
+until a disbursement mechanism for the lockbox is implemented in a future
+network upgrade. Such a disbursement mechanism should be designed to include an
+in-protocol option for key rotation, such that it is not necessary to perform a
+network upgrade to recover from key loss or compromise, or to change the size
+of the signing set or the number of signatures required to reach threshold.
+
+#### Security implications for Option 2a
```

Comment:
"for Mechanism 2a".

Reply:
Addressed in https://github.com/zcash/zips/pull/994